reader/src/main/java/org/kaazing/monitoring/reader/api/GatewayCounters.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I can't find any usage in  the viewer and statsD publisher. Do you have some local changes on your machine that you have not pushed into your forked repository on gitub? My understanding of the code I can see from your fault repository is that MMFReader is the main public interface for reading metrics, and it  provides methods getGatewayId and getCounters for getting the Gateway ID and counters, which seems fine to me. So I am not seeing why we need the GatewayCounters object.